### ISSUE TITLE
3 "Unchecked runtime.lastError: Cannot create item with duplicate id" errors in 9.1

### ISSUE THREAD

**Issue by MagicAndre1981** [PoziWorld/Scroll-To-Top-Button-extension#13]
I got the update to 9.1 and now Edge shows 3 "Unchecked runtime.lastError: Cannot create item with duplicate id":

![image](https://user-images.githubusercontent.com/8274816/73464140-baf85a80-437e-11ea-9ce9-04c522b06ead.png)

**Comment by PoziWorld** [comment]
Thank you for bringing this up.

Do you see the Scroll To Top Button menu when you right-click (open context menu) anywhere on a page?
If so, then this is not a critical issue, it shouldn't prevent anything from working.

But will still look into how to avoid it.

**Comment by MagicAndre1981** [comment]
Yes, I see the entry:

![image](https://user-images.githubusercontent.com/8274816/73518174-e706f080-43fd-11ea-8ce1-93a5481acd52.png)

**Comment by MagicAndre1981** [comment]
after a few days, the issue is gone:

![image](https://user-images.githubusercontent.com/8274816/73674750-77209080-46b1-11ea-9d55-138304007f0f.png)

I only see the 1 warning I posted some time ago.

**Comment by PoziWorld** [comment]
> after a few days, the issue is gone

Thank you for the follow-up, @MagicAndre1981!
It's still possible it'll come back up until a proper check is added in the code. Let's keep the issue open until it's added.

> I only see the 1 warning I posted some time ago.

Yeah... This one will keep popping up until Chromium gets native support for all locales the extension is translated into.

**Comment by MagicAndre1981** [comment]
this error is also not available in 9.3, but a new one:

```
Uncaught (in promise) TypeError: Cannot read property 'push' of undefined

Kontext

chrome-extension://chinfkfmaefdlchhempbfgbdagheknoj/_generated_background_page.html

Stapelüberwachung

global/js/background.js:128 (Anonyme Funktion)
```

![image](https://user-images.githubusercontent.com/8274816/75526393-fc2d6a00-5a11-11ea-8824-0b9a23722790.png)

**Comment by PoziWorld** [comment]
Will look into it. 
Thank you, @MagicAndre1981!

**Comment by PoziWorld** [comment]
> this error is also not available in 9.3, but a new one:
> 
> ```
> Uncaught (in promise) TypeError: Cannot read property 'push' of undefined
> Kontext
> chrome-extension://chinfkfmaefdlchhempbfgbdagheknoj/_generated_background_page.html
> Stapelüberwachung
> global/js/background.js:128 (Anonyme Funktion)
> ```
> ![image](https://user-images.githubusercontent.com/8274816/75526393-fc2d6a00-5a11-11ea-8824-0b9a23722790.png)

This one has been “muted” (commented-out for now) in v9.4.0.

**Comment by MagicAndre1981** [comment]
ok, now I only see the known sync issue, so I can close this here.
